### ISSUE TITLE
powershell_error when running vagrant halt

### ISSUE THREAD

**Issue by SalvesTestservices** [hashicorp/vagrant#10040]
### Vagrant version
Vagrant 2.1.2

### Host operating system
Windows 10

### Guest operating system
Ubuntu 16.04

### Vagrantfile
```ruby
Vagrant.configure("2") do |config|
  config.vm.box = "ubuntu/xenial64"
  config.vm.network :forwarded_port, guest: 3000, host: 3000
end
```
### Debug output
```ruby
==> default: Attempting graceful shutdown of VM...
    default:
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default:
    default: Inserting generated public key within guest...
translation missing: en.vagrant_ps.errors.powershell_error.powershell_error
```
### Expected behavior
Stopping the VM

### Actual behavior
The above error

### Steps to reproduce
vagrant halt

**Comment by chrisroberts** [comment]
Hi there. Would you please provide a gist of the output from the command with the `--debug` flag enabled. Thanks!

**Comment by chrisroberts** [comment]
Thanks! This was fixed in #9923 and is in the Vagrant 2.1.3 release. Upgrading will resolve this issue. Cheers!
